### ISSUE TITLE
Specify project folder and icon

### ISSUE THREAD

**Issue by dxcore35** [bgoosman/obsidian-project-garden#1]
Let user chose where the project folder is located. Also let set all other paramteres as icons, etc.. for user.

**Comment by danieltomasz** [comment]
Yes, it would be really useful to have the ability to specify the path and name to a folder with projects

**Comment by bgoosman** [comment]
Thanks for the comments!

**Comment by AaronKnowls** [comment]
I'm also interested in this. Unfortunately I can't use this plugin until I can edit the default projects folder since it's already tied to many dataview blocks. 
Thanks, bgoosman.

**Comment by Casyfill** [comment]
Same - love plugin, can't use
